### PR TITLE
UIIN-2482 Prevent editing of shared settings from outside "Consortium manager"

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIIN-2482

Prevent editing or deleting controlled vocabulary settings with the "consortium" source.

Note: applied for the settings from the [list](https://wiki.folio.org/display/FOLIJET/Consortium+manager+settings)

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by NikitaSedyx** [comment]
@usavkov-epam tests are failed

**Comment by usavkov-epam** [comment]
> @usavkov-epam tests are failed

Yeah, looking at them, but it seems fails are not connected with changes. For the master branch locally the same issue

File: src/settings/CallNumberTypes.test.js [hunk 1]
```diff
@@ -43,12 +46,11 @@ describe('CallNumberTypes', () => {
...
     it('should hide the "edit" and "delete" buttons', () => {
+      getSourceSuppressor.mockClear().mockReturnValue(() => true);
       renderCallNumberTypes();
 
-      const actionSuppressor = {
-        delete: expect(utils.sourceSuppressor).toHaveBeenCalledWith('system'),
-        edit: expect(utils.sourceSuppressor).toHaveBeenCalledWith('system'),
-      };
+      const suppressor = getSourceSuppressor(RECORD_SOURCE.SYSTEM);
+      const actionSuppressor = { delete: suppressor, edit: suppressor };
```
[the remark this answers is not in the document]
Comment (usavkov-epam) [reply src/settings/CallNumberTypes.test.js]: Thanks for noticing that, @Dmytro-Melnyshyn! Updated.

File: src/settings/CallNumberTypes.test.js [hunk 2]
```diff
@@ -4,17 +4,20 @@ import { MemoryRouter } from 'react-router-dom';
...
 } from '../../test/jest/helpers';
-
+import { RECORD_SOURCE } from '../constants';
 import CallNumberTypes from './CallNumberTypes';
 
+jest.mock('@folio/stripes/util', () => ({
+  ...jest.requireActual('@folio/stripes/util'),
+  getSourceSuppressor: jest.fn(),
+}));
 jest.mock('../utils');
```
[the remark this answers is not in the document]
Comment (usavkov-epam) [reply src/settings/CallNumberTypes.test.js]: Updated.

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**